requirements/edx/github.txt

```diff
@@ -92,6 +92,7 @@ git+https://github.com/edx/xblock-utils.git@v1.0.5#egg=xblock-utils==1.0.5
 git+https://github.com/edx/edx-user-state-client.git@1.0.1#egg=edx-user-state-client==1.0.1
 git+https://github.com/edx/xblock-lti-consumer.git@v1.1.5#egg=lti_consumer-xblock==1.1.5
 git+https://github.com/edx/edx-proctoring.git@0.18.2#egg=edx-proctoring==0.18.2
+git+https://github.com/edx/edx-val.git@1674ff42adcce7168f63acc16a44d2ef7f0e986f#egg=edxval==0.0.17
```

Comment:
Note: This is just for sandbox and for jenkins tests. Remove this before merging PR.